hs_discover/src/components/Resources.vue

```diff
@@ -452,6 +453,7 @@ export default {
           this.searchClick(false, false, false);
           this.setAllMarkers();
         } else {
+          this.setAllMarkers();
           recenterMap(); // eslint-disable-line
```

Comment:
Correct a workflow, whereby you enter maps, leave and come back after creating filters with no results. Adding this call re-renders markers (in this case having the effect of removing/clearing stale markers)